2016/proposals/narayan-aditya-visualizer.md

```diff
@@ -79,13 +87,135 @@ number of API calls left and the time of the day.
 
    -- DonaldKnuth
 
-### Database - PostgreSQL
+### Database - Django ORM
 
-The Django app will use PostgreSQL or SQLite as its
-database engine in production and SQLite database engine during
-development to store the survey responses fetched by the
+The SurveyMonkey Django app will use five major tables
+to store the survey responses fetched by the
 SurveyMonkey Django application.
 
+* **Survey**: The *Survey* table will consists of surveys fetched
+   and auto-populated by the [get_survey_details] API endpoint.
+* **Respondent**: The *Respondent* table stores basic user details
+  of all respondents for a given survey.
+* **Question**: Each survey row in the *Survey* table consists of
+  a list of questions, ranging from but not limited to
+  * Single choice / Dropdowns
+  * Matrix choice
+  * Text Box
+* **Answer**: Each question in the *Question* table has certain
+  answers that the respondent can select or give as an input. For eg:
+  * Single choice / Dropdowns: These questions have a list of answers
+    to them.
+  * Matrix choice: These questions have a row and a column of answers
+    to them.
+  * Text Box: The answer to these questions is a string.
+* **Response**: Each row in the *Response* table associates a respondent
+  with the question that was asked and the answer that
+  the respondent selected or inputted.
+
+I propose four database schemas.
```

Comment:
I was thinking about "the difference between these proposals lies in the method used to store respondents' answers …" or similar.